app/controllers/employers/census_employees_controller.rb

```diff
@@ -57,9 +57,16 @@ def edit
     @census_employee.benefit_group_assignments.build unless @census_employee.benefit_group_assignments.present?
   end
 
+  # path helper for employeeroster datatatble
   def show_employee
     @employee_id = params[:census_employee_id]
-   @census_employee = CensusEmployee.find(params[:census_employee])
+    @census_employee = CensusEmployee.find(params[:census_employee])
+  end
+
+  # path helper for employeeroster datatable
+  def rehire_employee
```

Comment:
The helper method looks good to me, I just need to assign  @census_employee. I will update it.